yarn-project/types/tsconfig.json

```diff
@@ -16,5 +16,5 @@
       "path": "../l1-artifacts"
     }
   ],
-  "include": ["src"]
+  "include": ["src", "../circuits.js/src/structs/two_field_hash.ts"]
```

Comment:
Should we consider moving it into types instead? It is circuit related, so might not be needed but this feels a bit clunky.